framework/db/mysql/Schema.php

```diff
@@ -174,7 +174,7 @@ protected function loadColumnSchema($info)
         $column->phpType = $this->getColumnPhpType($column);
 
         if (!$column->isPrimaryKey) {
-            if ($column->type === 'timestamp' && $info['default'] === 'CURRENT_TIMESTAMP') {
+            if (($column->type === 'timestamp' || $column->type ==='datetime') && $info['default'] === 'CURRENT_TIMESTAMP') {
```

Comment:
The DEFAULT CURRENT_TIMESTAMP support for a DATETIME (datatype) was added in MySQL 5.6. See details here: https://dev.mysql.com/doc/refman/5.6/en/timestamp-initialization.html .

And notice that the code both check column type and the default value, so even in the older version mysql, though column type is `datetiem`, but the second condition can't be true, so the code is compatible.

I have a question about unit test, base on the test database generated from `tests/data/mysql.sql`, I can't cover the case of this situation. How can I write unit test using the test database?